examples/scanner.py

```diff
@@ -19,6 +20,8 @@
     if platform.system() != "Darwin"
     else "B9EA5233-37EF-4DD6-87A8-2A875E821C46"  # <--- Change to your device's address here if you are using macOS
 )
+if len(sys.argv) == 2:
+    address = sys.argv[1]
```

Comment:
:+1: for adding this to all of the examples that take an address

Reply:
Done, insofar as I had the possibility to do at least minimal testing.